### DISCUSSION TITLE
500 errors for several people using visual studio code codespace

### DISCUSSION THREAD

**Discussion by AlexandraWeiner** [community/community#30290] [Other Feature Feedback, Questions, & Ideas]
Hello - I am running into a 504 error when I try to start the server using both `flask run` and `http-server`. I am taking a class, cs50, and others are getting 503 and 500 errors. This issue started last night. We have tried different browsers, clearing caches, restarting computers, creating new codespaces. Nothing seems to be resolving.

**Comment by tungnguyensnk** [comment]
good luck

**Comment by ryanprograming** [comment]
i am going through the same stuff

**Reply by ahmedjahmi** [reply]
has this been answered? having the same problem now.

**Comment by AsafFisher** [comment]
Same
